articles/overview/what-is-qsharp-and-qdk.md

```diff
@@ -0,0 +1,66 @@
+---
+title: What is the Q# programming language and QDK?
+description: Learn about the Microsoft Quantum Development Kit, the Q# programming language, and how you can create quantum programs.
+author: bradben
+ms.author:  bradben
+ms.date: 04/22/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.q-sharp
+---
+
+# What is the Q# programming language and QDK?
+
+Q# is Microsoft’s open-source programming language for developing and running quantum algorithms. It’s part of the Quantum Development Kit (QDK), which includes Q# libraries, quantum simulators, extensions for other programming environments, and API documentation. In addition the Standard Q# library, the QDK includes Chemistry, Machine Learning, and Numeric libraries.
+
+As a programming language, Q# draws familiar elements from Python, C#, and F# and supports a basic procedural model for writing programs with loops, if/then statements, and common data types. It also introduces new quantum-specific data structures and operations.
+
+## What can I do with the QDK?
+
+The QDK is a full-featured development kit for Q# that you can use with common tools and languages to develop quantum applications that you can run in various environments.
```

Comment:
It may help to note that you can use Q# on its own, either through Jupyter Notebooks or to write command-line apps, as well as part of Python and/or .NET workflows.